components/sup/src/event.rs

```diff
@@ -50,14 +48,14 @@ lazy_static! {
     // this lazy_static call.
 
     // NATS subject names
-    static ref SERVICE_STARTED_SUBJECT: Subject =
-        "habitat.event.service_started".parse().expect("valid NATS subject");
-    static ref SERVICE_STOPPED_SUBJECT: Subject =
-        "habitat.event.service_stopped".parse().expect("valid NATS subject");
-    static ref SERVICE_UPDATE_STARTED_SUBJECT: Subject =
-        "habitat.event.service_update_started".parse().expect("valid NATS subject");
-    static ref HEALTHCHECK_SUBJECT: Subject =
-        "habitat.event.healthcheck".parse().expect("valid NATS subject");
+    static ref SERVICE_STARTED_SUBJECT: String =
+        "habitat.event.service_started".to_string();
```

Comment:
since these are just strings, you can now move out of the `lazy_static!` replace with:

```
const SERVICE_STARTED_SUBJECT: &str = "habitat.event.service_started";
```
Same for the other subjects below.

Reply:
OK - will do!

Reply:
Updated